src/Content/VersionRules.php

```diff
@@ -68,7 +68,8 @@ public static function delete(
 		Language $language
 	): void {
 		if ($version->isLocked('*') === true) {
-			throw new LogicException(
+			throw new LockException(
+				lock: $version->lock('*'),
 				message: 'The version is locked and cannot be deleted'
```

Comment:
`Question:` Should we add i18n strings for these. And then we could actually use the exception message in the lock dialog.